src/app/wishlists/wishlist-fetch.ts

```diff
@@ -0,0 +1,64 @@
+import { toWishList } from './wishlist-file';
+import { t } from 'app/i18next-t';
+import _ from 'lodash';
+import { showNotification } from 'app/notifications/notifications';
+import { loadWishLists } from './actions';
+import store from 'app/store/store';
+
+export function fetchWishList(showAlert: boolean) {
```

Comment:
This is a good candidate for a thunk action - an action that you can dispatch which is a function that takes `(dispatch, getState)` and can then dispatch any number of actions asynchronously. This avoids you having to explicitly grab the global `store` reference.

Reply:
The approach is a bit gnarly, but it's in there.